src/view/ChatView.java

```diff
@@ -34,6 +44,35 @@ public ChatView(ChatController chatController, ChatViewModel viewModel) {
     this.add(scrollPane);
     this.add(messagePanel);
 
+    send.addActionListener(
+      // This creates an anonymous subclass of ActionListener and instantiates it.
+      new ActionListener() {
+        public void actionPerformed(ActionEvent evt) {
+          if (evt.getSource().equals(send)) {
+            ChatState currentState = viewModel.getState();
+
+            Message newMessage = new Message(Instant.now(), currentState.message);
+            chatController.sendMessage(newMessage);
```

Comment:
I wonder if the UCI should handle the Instant.now construction? (this isn't necessary, but it seems like the view doesn't really need to care about time)

Reply:
Fixed by refactoring `Instant.now()` into the use case interactor